types/FingerTable.go

```diff
@@ -0,0 +1,14 @@
+package types
+
+import "fmt"
+
+type FingerEntry struct {
+	Id   []byte
+	GossiperAddress string
+}
+
+type FingerTable []*FingerEntry
```

Comment:
Do you mind using as few pointers as possible? Having pointers means that it may be null and you have to check for nullity all the time. It also means that you need to *think about* cases of what happens when there *is* a null.